util/get-toolchain.py

```diff
@@ -115,9 +115,15 @@ def install(archive_file, target_dir):
     subprocess.run(cmd, check=True)
 
 
-def postinstall_rewrite_configs(install_path):
-    """This rewrites the toolchain configuration files to point to install_path"""
-    if str(install_path) == TARGET_DIR:
+def postinstall_rewrite_configs(install_path, release_path):
+    """Rewrites the toolchain configuration files to point to release_path.
+
+    In most cases, the install_path and release_path are identical. However, in
+    corporate environments with centrally installed tools, the installation may
+    be done in a temporary staging area first (by the engineers who need it for
+    their work), before it is moved to the release area (managed by a separate
+    IT department)."""
```

Comment:
Can you make this explanation less of an example/motivation, but more of a description of what the paths actually are? E.g. install_path is the final installation directory from which the tools are called, release_path is a installation staging directory, from where the tools are expected to be manually moved to install_dir.